<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Style_DW</class>
 <widget class="QMainWindow" name="Style_DW">
  <property name="windowTitle">
   <string>Style Sheets</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>399</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>14</number>
       </property>
       <property name="rightMargin">
        <number>14</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item row="1" column="2" colspan="2">
        <widget class="QRadioButton" name="femaleRadioButton">
         <property name="text">
          <string>&amp;Female</string>
         </property>
         <property name="toolTip">
          <string>Check this if you are female</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QListWidget" name="professionList">
         <property name="currentRow">
          <number>-1</number>
         </property>
         <property name="whatsThis">
          <string>Specify your name here</string>
         </property>
         <property name="toolTip">
          <string>Select your profession</string>
         </property>
         <property name="statusTip">
          <string>Specify your name here</string>
         </property>
         <item>
          <property name="text">
           <string>Computer Programmer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dancer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Flight Attendant</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Professor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Student</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yoga Teacher</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Profession:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>professionList</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="countryLabel">
         <property name="text">
          <string>Country:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>countryCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer">
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>15</width>
           <height>8</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="ageSpinBox">
         <property name="value">
          <number>22</number>
         </property>
         <property name="minimum">
          <number>12</number>
         </property>
         <property name="toolTip">
          <string>Specify your age</string>
         </property>
         <property name="statusTip">
          <string>Specify your age</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ageLabel">
         <property name="text">
          <string>&amp;Age:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>ageSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="genderLabel">
         <property name="text">
          <string>Gender:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>&amp;Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>nameCB</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="maleRadioButton">
         <property name="text">
          <string>&amp;Male</string>
         </property>
         <property name="toolTip">
          <string>Check this if you are male</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <spacer name="horizontalSpacer">
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="6">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="aboutPB">
           <property name="text">
            <string>About</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="editPB">
           <property name="text">
            <string> Edit/Select Style </string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closePB">
           <property name="text">
            <string>Close</string>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QComboBox" name="countryCombo">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="toolTip">
          <string>Specify country of origin</string>
         </property>
         <property name="statusTip">
          <string>Specify country of origin</string>
         </property>
         <item>
          <property name="text">
           <string>Brazil</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Canada</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>France</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Germany</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Italy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Norway</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>United States</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QComboBox" name="nameCB">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Specify your name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <spacer name="horizontalSpacer_5">
         <property name="sizeHint" stdset="0">
          <size>
           <width>61</width>
           <height>20</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Close Style Sheets</string>
   </property>
  </action>
  <action name="actionEditStyle">
   <property name="text">
    <string>Edit &amp;Style...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>nameCB</tabstop>
  <tabstop>maleRadioButton</tabstop>
  <tabstop>femaleRadioButton</tabstop>
  <tabstop>ageSpinBox</tabstop>
  <tabstop>countryCombo</tabstop>
  <tabstop>professionList</tabstop>
  <tabstop>aboutPB</tabstop>
  <tabstop>editPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
